name-segments/surname-orc.txt

```diff
@@ -0,0 +1,15 @@
+Araagnous
+Borg
+Blorg
+Clork
+Chungus
+Dungus
+Glorg
+Gutrack
+Gung
+Hoorge
+Horungus
+Skullump
+Woarg
+
+
```

Comment:
```suggestion
Araagnous
Borg
Blorg
Clork
Dungus
Glorg
Gordock
Gutrack
Gung
Hoorge
Horungus
Mogor
Skullump
Woarg
```
Shouldn't have any blank lines at the end of the file. (replaced them with two additional surnames)
I dropped Chungus for its meme origins: https://www.dictionary.com/e/slang/chungus/

Other than that, it looks good - thanks for contributing